lib/msal-common/src/response/SPAResponseHandler.ts

```diff
@@ -229,7 +229,7 @@ export class SPAResponseHandler {
 
         // Retrieve current id token object
         let idTokenObj: IdToken;
-        const cachedIdToken: string = this.cacheStorage.getItem(PersistentCacheKeys.ID_TOKEN);
+        const cachedIdToken: string = this.cacheStorage.getItem(PersistentCacheKeys.ID_TOKEN) as string;
```

Comment:
Is `as string` needed? Looks like `getItem` returns a string.

Reply:
If `getItem` can return multiple types, I would argue for generics instead of making consumers cast.

Reply:
This will be resolved in a future PR when we add functionality for consumer driven storage.